### PR TITLE
[DOCS] TabContainer class descriptions

### PR DESCRIPTION
**Pull request by brunosxs**

### CODE REVIEW

File: doc/classes/TabContainer.xml [hunk 1]
```diff
@@ -42,14 +42,14 @@
...
-				Return the previous tab index that was being shown.
+				Returns the previous tab index that was being shown.
 			</description>
 		</method>
 		<method name="get_tab_align" qualifiers="const">
 			<return type="int" enum="TabContainer.TabAlign">
 			</return>
 			<description>
-				Return tab alignment, from the ALIGN_* enum.
+				Returns the tab alignment, from the ALIGN_* enum.
```
Comment (cbscribe) [inline doc/classes/TabContainer.xml]: Change the , to a . and make the last sentence conform to the enum standard: See the ALIGN_* constants.

File: doc/classes/TabContainer.xml [hunk 2]
```diff
@@ -157,16 +160,19 @@
...
 	</methods>
 	<members>
 		<member name="current_tab" type="int" setter="set_current_tab" getter="get_current_tab">
+			The current tab.
 		</member>
 		<member name="tab_align" type="int" setter="set_tab_align" getter="get_tab_align" enum="TabContainer.TabAlign">
+			The alignment of all the tabs of the tab container. See [code]ALIGN_*[/code] constants.
 		</member>
 		<member name="tabs_visible" type="bool" setter="set_tabs_visible" getter="are_tabs_visible">
+			If [code]true[/code], all tabs that are children of the TabContainer will be visible.
```
Comment (cbscribe) [inline doc/classes/TabContainer.xml]: Remove the ,

**Comment by brunosxs** [comment]
Done.

File: doc/classes/TabContainer.xml [hunk 3]
```diff
@@ -157,16 +160,19 @@
 			<argument index="0" name="visible" type="bool">
 			</argument>
 			<description>
-				Set whether the tabs should be visible or hidden.
+				If [code]true[/code] all the tabes will be visible.
```
Comment (cbscribe) [inline doc/classes/TabContainer.xml]: tabes -> tabs

**Comment by brunosxs** [comment]
Sorry for the typo

**Comment by cbscribe** [comment]
Thanks for your contribution!

**Comment by brunosxs** [comment]
I am the one who has to thank you for the patience. I am learning git as I am doing things.
Hope I can help with c++ code in the futre.
Thanks.

**Pull request merged**